src/common/Card.ts

```diff
@@ -1,20 +1,77 @@
 import { getAnimations } from "../getStyles";
 import { encodeHTML, flexLayout } from "./utils";
 
+/** Card colors. */
+export interface CardColors {
+  /** Title color. */
+  titleColor: string;
+  /** Text color. */
+  textColor: string;
+  /** Icon color. */
+  iconColor: string;
+  /** Background color. */
+  bgColor: string;
+  /** Border color. */
+  borderColor: string;
+}
+
+/** Accessibility label. */
+interface AccessibilityLabel {
+  /** The label to display. */
+  title: string;
+  /** The value to display. */
+  desc: string;
+}
+
+/** Card properties. */
+interface CardProps {
+  /** Card width. */
+  width: number;
+  /** Card height. */
+  height: number;
+  /** Card border radius. */
+  border_radius: number;
+  /** Card colors. */
+  colors: CardColors | {};
+  /** Card title. */
+  customTitle?: string;
+  /** Card default title. */
+  defaultTitle?: string;
+  /** Card title prefix icon. */
+  titlePrefixIcon?: string;
+}
+
+/**
+ * Card class.
+ */
 export class Card {
+  /** Card width. */
   width: number;
+  /** Card height. */
   height: number;
+  /** Whether the card border is hidden. */
   hideBorder: boolean;
+  /** Whether the card title is hidden. */
   hideTitle: boolean;
+  /** Border radius. */
   border_radius: number;
-  colors: any;
-  title: any;
+  /** Card colors. */
+  colors: CardColors | {};
+  /** Card title. */
+  title: string;
+  /** Card css. */
   css: string;
+  /** Card x padding. */
   paddingX: number;
+  /** Card y padding. */
   paddingY: number;
-  titlePrefixIcon: string;
+  /** Card title prefix icon. */
+  titlePrefixIcon: string | undefined;
```

Comment:
How about change this line to ```titlePrefixIcon?: string;```
I would like to unify to your convention in src/common/utils.ts
```
secondaryMessage?: string;
```